.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,39 @@
+name: build-docker
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Deployign to the docker registry on pull_request is a security concern, only deploy master branch on push.